### DISCUSSION TITLE
Queue effects and UI effects for validation.

### DISCUSSION THREAD

**Discussion by DanielWillett** [SmartlyDressedGames/Unturned-3.x-Community#3700] [Ideas]
Effect conflicts can be common, especially with UI since they still work off ushorts. If a UI effect is meant to cover a full screen and the player is missing it, they will just be stuck on the screen with no way to close or otherwise interact. Adding effects and UI effects to ClientAssetIntegrity checking could help with this.

**Comment by SDGNelson** [comment]
Regular effects do use ClientAssetIntegrity, but your guess is right that UI effects don't because they still use uint16.
